src/BenchmarkDotNet.Runtime/Portability/RuntimeInformation.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using BenchmarkDotNet.Environments;
+
+namespace BenchmarkDotNet.Portability
+{
+    public abstract class RuntimeInformation
+    {
+        internal const string Unknown = "?";
+        internal const string DebugConfigurationName = "DEBUG";
+        internal const string ReleaseConfigurationName = "RELEASE";
+
+        public virtual bool IsMono => false;
+
+        public abstract bool IsWindows { get; }
+        public abstract bool IsLinux { get; }
+        public abstract bool IsMac { get; }
```

Comment:
I think it makes sense to introduce own `OS` enum instead of 3 boolean properties.